blackbox/docs/best_practice/migrating_from_mysql.txt

```diff
@@ -148,6 +148,24 @@ Finally use the ``COPY FROM`` statement to import your newly generated json file
   Read :ref:`efficient_data_import` for more information how to import huge datasets into Crate.
 
 
+3rd Party Tool: csvkit
+======================
+
+The Python tools provided by `csvkit`_ allow you to directly insert CSV data
+into Crate via SQLAlchemy, using Crate’s native driver to create the table,
+guess the corresponding data types, and insert any data found in the CSV file.
+
+The simplest way to do so is like this:
+
+.. code-block:: sh
+
+  $ csvsql --db crate://localhost:4200 --insert /tmp/dump.csv
```

Comment:
this requires to have the latest version of [`crate`](https://pypi.python.org/pypi/crate) installed and in your `$PYTHONPATH`, right?